src/pages/TradeV3/perps/components/PerpsGenericComp.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { cn } from 'gfx-component-lib'
+import { FC, ReactElement, ReactNode, useEffect, useRef, useState } from 'react'
+
+export const BlackGradientBg: FC<{ children: ReactNode }> = ({ children }) => (
+  <div
+    className="fixed bottom-0 left-0 w-full h-20  bg-gradient-2
+z-50 flex justify-center items-center"
+  >
+    {children}
+  </div>
+)
+
+export const InfoLabel: FC<{ children: ReactNode }> = ({ children }) => (
+  <div>
+    <h5 className={cn('text-black-4 dark:text-grey-8')}>{children}</h5>
+  </div>
+)
+export const GradientBorder: FC<{ children: ReactNode; radius: number }> = ({
+  children,
+  radius
+}): ReactElement => (
+  <div className={`w-full p-[1px] bg-gradient-1 rounded-[${radius}px] h-10`}>
+    <div className={`bg-grey-5 dark:bg-black-1 h-full rounded-[${radius}px]`}>{children}</div>
+  </div>
+)
+export const GradientButtonWithBorder: FC<{ children: ReactNode; radius: number; height: number }> = ({
```

Comment:
You don't need this, you can use `variant='outline'` `colorScheme='primaryGradient'` and it would do the same. Unless there is a reason for this?

Reply:
No, This one is needed, this has gradient background